### DISCUSSION TITLE
H4 и WM210

### DISCUSSION THREAD

**Discussion by Kosnov** [kachurovskiy/nanoels#211] [Show and tell]
Вот как получилось
![photo_2024-08-12_19-06-31](https://github.com/user-attachments/assets/ea769d49-b911-4748-bf71-bddd5eb9e2b3)
![photo_2024-08-12_19-06-38](https://github.com/user-attachments/assets/0ccda4ea-9fd2-4d81-bc3d-1f1b0626c542)
Видео работы

https://github.com/user-attachments/assets/04e35ad3-60b8-4606-8a7b-bec4035f01c3

https://github.com/user-attachments/assets/79db5f02-b810-4242-b369-f30147fe1f1d

Максим, благодарю за проект. Успехов.

**Comment by kachurovskiy** [comment]
Классно! Спасибо что прислали фото и видео.

**Comment by Kosnov** [comment]
Всем доброго.
Это новый дизайн.
![photo_2024-08-17_17-00-30](https://github.com/user-attachments/assets/3f49229b-3b35-46fb-8ddf-d7d05c539c14)
![photo_2024-08-17_17-00-41](https://github.com/user-attachments/assets/903c082d-586f-451d-852c-475ee07fbea3)

**Comment by tektonandroid** [comment]
Nos gustaría ver tu instalación de motores....Al menos a mi. 😅

**Reply by Kosnov** [reply]
Definitivamente tomaré una foto

**Comment by Kosnov** [comment]
![7BB0194B-E5A6-4B9F-92B5-2F80DB5C2169](https://github.com/user-attachments/assets/13ed8f19-9fe9-410a-ac2b-1daee7ebe366)
![F742994C-1172-44AE-9E4F-325DF90252D4](https://github.com/user-attachments/assets/b421c5c7-b89e-4c4e-bc4c-9100b8ff0d5f)
![F08D5062-23E7-4EF2-B1C7-8C067B8E0900](https://github.com/user-attachments/assets/b5f554b4-6eda-4ad4-81fe-c572fd631b29)
![EFE5526C-7C3E-4EC8-A9FC-EE6B8522B999](https://github.com/user-attachments/assets/5ecb65a6-d802-497f-bed1-372a73f7739d)
![A1E7EC76-F636-4351-B02C-669E05673C96](https://github.com/user-attachments/assets/79806c4f-3472-4fc1-9a18-b3e015a309d9)

**Reply by kachurovskiy** [reply]
Классно что мотор на основную ось получилось соосно разместить.

К сожалению пластик обычно проседает под винтами и ремень ослабляется. 

На энкодер попробуй ремень GT3, идеально подходит, тихо работает, гибкий и зубы не слишком маленькие, хорошо по печатным роликам ходят.

**Reply by Kosnov** [reply]
Доброго Максим. Сейчас на переделке вот этот станок. Тут на энкодер ремень будет однозначно GT2. 
![6A4A588D-20B0-4A00-926A-BD2FBFAA844C](https://github.com/user-attachments/assets/f2baad3a-4668-449f-b4fd-7ad723925d5d)

**Reply by kachurovskiy** [reply]
GT2 по мне зубы мелковаты - но если работает то отлично 👍

**Comment by Kosnov** [comment]
![241B715C-4C9C-4C1C-B446-B44229733859](https://github.com/user-attachments/assets/3e8b57d1-8794-45de-8cb1-d224ad229abf)

**Comment by Viktor1972-avi** [comment]
Еще одна история
![20241008_203409](https://github.com/user-attachments/assets/d7ba656f-0262-401e-a2a5-91c2ae3b1cac)
![20241008_203419](https://github.com/user-attachments/assets/011700fd-01e0-402c-a461-6ac7fe43154d)
![20241008_203424](https://github.com/user-attachments/assets/f95a40ae-c580-4206-b295-d10cdc999aea)
![20241008_203442](https://github.com/user-attachments/assets/4ed3f67a-87fa-472c-9638-ca7353176a49)
![20241008_203528](https://github.com/user-attachments/assets/21405340-324a-46e4-bb95-6a2a50bce36b)
![20241008_203534](https://github.com/user-attachments/assets/1dfc380d-9882-4d01-8d80-767d3e2da331)
![20241008_203540](https://github.com/user-attachments/assets/e1b2a694-19bd-4410-b90f-a3741af9da15)
![20241008_203548](https://github.com/user-attachments/assets/a274bedb-ccd1-4592-bf52-ca174544ebcf)

**Reply by kachurovskiy** [reply]
Обалдеть! Поделитесь моделями, схемами и кодом?

**Reply by Charus-OV** [reply]
Это только ты делишься )))
